### PR TITLE
FTV-514: Crash with activation

### PR DESCRIPTION
**Pull request by clusty**
context was being changed before the jobs that depended on it were finished.

### CODE REVIEW

File: com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs [hunk 1]
```diff
@@ -62,26 +116,25 @@ public static void ReconnectStreamsWithPath(string path)
         AlembicStreamDescriptor m_streamDesc;
         AlembicTreeNode m_abcTreeRoot;
         aiConfig m_config;
-        aiContext m_context;
+        ContextPair context;
```
Comment (judubu) [inline com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs:119]: Why the name change? m_context to context

File: com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs [hunk 2]
```diff
@@ -9,15 +9,69 @@
...
 {
-    internal sealed class AlembicStream : IDisposable
+    sealed class AlembicStream : IDisposable
     {
-        public struct UpdateSamplesJob : IJob
+        /// <summary>
+        /// This class hides the context. The reason for that it that there are Jobs depending on it and we need
+        /// to ensure they get completed before any set happens on the context
+        /// </summary>
+        public struct ContextPair
```
Comment (judubu) [inline com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs:18]: Can I suggest SecureContext or SyncedContext? I think it conveys more meaning than ContextPair

File: com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs (hunk 1 above)
Comment (clusty) [reply com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs]: I hate variables starting with m :) Fixed for consistency.

File: com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs (hunk 2 above)
Comment (clusty) [reply com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs]: changed to SafeContext.

**Pull request merged**